### PR TITLE
Separated Lint tests

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,16 +4,18 @@ python:
   - "2.7"
 
 env:
-  - VERSION="8.0" ODOO_REPO="odoo/odoo"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB"
-  - VERSION="8.0" UNIT_TEST="1"
+  - VERSION="8.0" LINT_CHECK="1"
```
Comment (pedrobaeza) [inline .travis.yml:7]: Why 2 envs?

Comment (yvaucher) [inline .travis.yml:7]: Good question

Comment (pedrobaeza) [inline .travis.yml:7]: I think you can remove one of them

**Comment by yvaucher** [comment]
@pedrobaeza I was wondering which PR it was already. Done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by tafaRU** [comment]
:+1:

**Comment by dreispt** [comment]
:+1:

**Pull request merged**